apps/dashboard/src/components/create-workflow-button.tsx

```diff
@@ -128,6 +129,7 @@ export const CreateWorkflowButton = (props: CreateWorkflowButtonProps) => {
                         <Input
                           {...field}
                           autoFocus
+                          data-1p-ignore
```

Comment:
Adding support for Dashlane (ref: https://support.dashlane.com/hc/en-us/articles/4420122792594-Optimize-your-web-forms-for-Dashlane-Autofill#:~:text=data%2Dform%2Dtype%3D%E2%80%9Dother%E2%80%9D).

Perhaps we should create an `AUTOCOMPLETE_OFF` constant to store all of these for share use across the FE?

```suggestion
                          data-1p-ignore
                          data-form-type="other"
```